pvlib/temperature.py

```diff
@@ -978,3 +978,397 @@ def prilliman(temp_cell, wind_speed, unit_mass=11.1, coefficients=None):
+        net_absorptance_pvsyst = self.alpha * (1.0 - self.eta)
+        absorptance_ratio = net_absorptance_glm / net_absorptance_pvsyst
+
+        u_c = self.u_const / absorptance_ratio
+        u_v = self.du_wind / absorptance_ratio
+
+        return dict(u_c=u_c,
+                    u_v=u_v,
+                    module_efficiency=self.eta,
+                    alpha_absorption=self.alpha)
+
+    def use_noct_sam(self, noct, module_efficiency=None,
+                     transmittance_absorptance=None):
+        '''
+        Use the NOCT SAM model parameters to set the generic equivalents.
+
+        Parameters
+        ----------
+        noct : float
+            See :py:func:`pvlib.temperature.noct_sam` for details.
+
+        module_efficiency, transmittance_absorptance : float, optional
+            See :py:func:`pvlib.temperature.noct_sam` for details.
+
+        Notes
+        -----
+        The optional parameters are primarily for convenient compatibility
+        with existing function signatures.
+        '''
+        if module_efficiency is not None:
+            self.eta = module_efficiency
+
+        if transmittance_absorptance is not None:
+            self.alpha = transmittance_absorptance
+
+        # NOCT is determined with wind speed near module height
+        # the adjustment reduces the wind coefficient for use with 10m wind
+        wind_adj = 0.51
+        u_noct = 800.0 * self.alpha / (noct - 20.0)
+        self.u_const = u_noct * 0.6
+        self.du_wind = u_noct * 0.4 * wind_adj
+        return self
+
+    def to_noct_sam(self):
+        '''
+        Convert the generic model parameters to NOCT SAM model equivalents.
+
+        Returns
+        ----------
+        model_parameters : dict
+            See :py:func:`pvlib.temperature.noct_sam` for
+            model parameter details.
+        '''
+        # NOCT is determined with wind speed near module height
+        # the adjustment reduces the wind coefficient for use with 10m wind
+        wind_adj = 0.51
+        u_noct = self.u_const + self.du_wind / wind_adj
+        noct = 20.0 + (800.0 * self.alpha) / u_noct
+        return dict(noct=noct,
+                    module_efficiency=self.eta,
+                    transmittance_absorptance=self.alpha)
+
+    def use_sapm(self, a, b, wind_fit_low=1.4, wind_fit_high=5.4):
+        '''
+        Use the SAPM module model parameters to set the generic equivalents.
+
+        In the SAPM the heat transfer coefficient increases exponentially
+        with windspeed, whereas in the other models the increase is linear.
+        This function equates the generic linear model to SAPM at two
+        specified winds speeds, thereby defining a linear approximation
+        for the exponential behavior.
+
+        Parameters
+        ----------
+        a, b : float
+            See :py:func:`pvlib.temperature.noct_sam` for details.
+
+        wind_fit_low, wind_fit_high : float, optional
```

Comment:
I think we want to list these separately. The docstring parser renders them as two different parameters with the same description "Two wind speed values..."